cdap-docs/_common/common-build.sh

```diff
@@ -511,13 +490,10 @@ function run_command() {
     check-includes )    check_includes;;
     docs )              build_docs;;
     license-pdfs )      build_license_pdfs;;
-    build-standalone )  build_standalone;;
     copy-javadocs )     copy_javadocs;;
     copy-license-pdfs ) copy_license_pdfs;;
-    javadocs )          build_javadocs_sdk;;
+    javadocs-api )      build_javadocs_api;;
     javadocs-full )     build_javadocs_full;;
-    depends )           build_dependencies;;
-    sdk )               build_sdk;;
     version )           display_version;;
     * )                 usage;;
```

Comment:
This whole section could be simpler if you had a 1:1 command:function naming... perhaps with `s/-/_/` applied...

fex.
```
check-includes|docs|copy-javadocs|copy-license-pdfs) ${1/-/_};;
*) usage;;
```